package.json

```diff
@@ -18,8 +18,7 @@
     "lint:watch": "watch 'npm run lint --silent' src test",
     "prepublish": "npm run build",
     "posttest": "npm run lint --silent",
-    "test": "mocha",
-    "test:coverage": "nyc --reporter=text --reporter=lcov mocha",
+    "test": "nyc --reporter=text --reporter=lcov mocha",
```

Comment:
No more `test:coverage`, we'll save that for actually [integrating with codecov.io](https://github.com/bcoe/nyc#integrating-with-codecov).  We now show coverage after every test run.

`posttest` was not suitable for this since coverage must be generated before reporting it.  So, we'd end up running tests without coverage, then running again to generate coverage and report it.  Opted for the simpler "always generate coverage" route.